include/gpd/grasp_image_15_channels.h

```diff
@@ -34,7 +34,7 @@
 #define GRASP_IMAGE_15_CHANNELS_H_
 
 
-#include "../gpd/grasp_image.h"
+#include <gpd/grasp_image.h>
```

Comment:
To include a header file from "local" project, it's recommended to use '#include "gpd/xxx.h"', instead of <>.